youtube_dl/extractor/openload.py

```diff
@@ -140,7 +140,7 @@ def __del__(self):
         for name in self._TMP_FILE_NAMES:
             try:
                 os.remove(self._TMP_FILES[name].name)
-            except:
+            except (OSError, TypeError):
```

Comment:
What's the point of `TypeError` here?

Reply:
`os.remove()` throws TypeError if the passed file path is `None`; the original implementation with a bare `except` would've caught that, so including it was in the interest of not breaking anything while making the code flake8-compliant

Reply:
`self._TMP_FILES[name].name` can't be `None`.

Reply:
How so? Sorry, I'm no Python guru - is that some property of dictionaries (or whichever type `self._TMP_FILES` is)?

Reply:
Ah, I see, `_TMP_FILES` is a dictionary, but it only contains `tempfile.NamedTemporaryFile` instances, so it should be safe to assume that `self._TMP_FILES[name].name` won't ever be none.. Okay, I'll push a fix.